payments_service/braintree/templates/braintree/emails/subscription_canceled.html

```diff
@@ -0,0 +1,37 @@
+{# subscription header is added automatically here #}
+
+    <div class="padded">
+        <button disabled>Subscription cancelled</button>
+
+        <table>
+            <tr class="cc">
+                <th scope="row">Paid with</th>
+                {# __CC_TYPE__ is post-process replaced #}
+                <td><img class="card" alt="__CC_TYPE__" src="/svg/__CC_TYPE__.svg" width="40" /> ending in {{ cc_truncated_id }}</td>
+            </tr>
+            <tr class="desc">
+                <th scope="row">{{ product.description }}</th>
+                <td>${{ moz_trans.amount }} <span>per month</span></td>
+            </tr>
+        </table>
+
+        <div class="divider"></div>
+
+        <table class="total-amount">
+            <tr class="total">
+                <th scope="row">Total</th>
+                <td>${{ moz_trans.amount }}</td>
+            </tr>
+            <tr>
+                <th>Active until</th>
+                <td>{{ bill_end }}</td>
+            </tr>
+        </table>
+    </div>
+
+    <div class="footer">
```

Comment:
Oh I see this is separate block - this is probably OK as is I guess `<footer>` is probably not supported by some email clients so playing it safe is probably wise.